labs/fabric-netkit.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Fabric Netkit
+parent: Labs
+---
+
+# Lab Name
+
+[Fabric Netkit](https://github.com/hyperledger-labs/fabric-netkit)
+
+# Short Description
+
+Software development kit for access and management of Hyperledger Fabric network
+
+# Scope of Lab
+
+The scope of this lab is to deliver the software development kit for access and management Hyperledger Fabric network:
+
+* query and invoke chaincodes, using local configuration and service discovery
+* sdk for interacting with system chaincodes (csss, lscc, qscc, lifecycle)
+* deliver and parse blocks
+* build, install and instantiate chaincodes
+* configure channels
+
+# Initial Committers
+
+- https://github.com/vitiko
+- https://github.com/dimaxgl
+- https://github.com/bogatyr285
+- https://github.com/dimuls
+- https://github.com/inotnako
+- https://github.com/criro1
+
+# Sponsor
+
+- https://github.com/C0rWin - HLF maintainer, Hyperledger TSC Member
+
+# Pre-existing repository
+
+- https://github.com/s7techlab/hlf-sdk-go
```

Comment:
It looks like the commit in this repository does not contain DCO signoff. Hyperledger requires that all contributed code has a DCO sign off. Looking at these repositories, it appears that this is not the case. There are two options that we normally consider for fixing this:

1. Amend existing commits in the repository to include DCO sign off
1. Manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs

Reply:
I prefer option 2